### PR TITLE
fix: og:image meta property not an absolute path

### PR DESCRIPTION
**Pull request by EnumC**
og:image should point to an absolute path.

This PR replaces the relative path with an absolute path.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1">
-
+    
     <meta property="og:title" content="Bananium Labs">
     <meta property="og:description" content="Official website for Bananium Labs">
     <meta property="og:type" content="website">
     <meta property="og:url" content="https://bananium.com">
-    <meta property="og:image" content="img/favicon.ico">
-    <meta property="og:site_name" content="Bananium Labs">
+    <meta property="og:image" content="https://bananium.com/img/banner-v5.png">
```
Comment (64bitpandas) [inline dist/index.html:16]: Why use https instead of doing just `/img/banner-v5.png`?

**Review by 64bitpandas — APPROVED** [review]
lgtm, have question tho see comments

File: dist/index.html (hunk 1 above)
Comment (EnumC) [reply dist/index.html:16]: Looks to me the opengraph spec doesn't allow for relative paths. They also got recommended image size, so we might have to resize the image whenever we're free.

Comment (64bitpandas) [reply dist/index.html:16]: ok cool, sounds good. maybe make an issue for that so we don't forget?

you can merge this in now tho

**Pull request merged**